website/addons/dataverse/model.py

```diff
@@ -71,6 +72,24 @@ def dataset_id(self, value):
     def complete(self):
         return bool(self.has_auth and self.dataset_doi is not None)
 
+    @property
+    def folder_id(self):
+        return self.dataset_id
+
+    @property
+    def folder_path(self):
+        pass
```

Comment:
It's a property of the `AddonOAuthNodeSettingsBase` superclass that "needs" to be implemented, but Dataverse doesn't behave like a normal storage addon, so it's not actually used.